ironfish/src/memPool/memPool.ts

```diff
@@ -115,17 +116,53 @@ export class MemPool {
     await this.feeEstimator.init(this.chain)
   }
 
+  /**
+   *
+   * @returns The number of transactions in the mempool
+   */
   count(): number {
     return this.transactions.size
   }
 
+  /**
+   *
+   * @returns The maximum number of transactions that can be stored in the mempool.
+   *
+   * This assumes that all transactions are the maximum size, and so the true
+   * maximum in practice can be higher.
+   */
+  maxCount(): number {
```

Comment:
Are we using this anywhere?

Reply:
this is used for for the max capacity and saturation recently evicted cache metrics